fuzzing/libfuzzer/conftest.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+import os
+from libs import utils
+from libs import gramine_libs
+from common.config.constants import *
+
+@pytest.fixture(scope="session", autouse=True)
+def gramine_setup():
+    print_env_variables()
+    print("\n###### In gramerf_setup #####\n")
+
+    # Delete old logs if any and create new logs directory.
+    if os.path.exists(LOGS_DIR):
+        del_logs_cmd = 'rm -rf ' + LOGS_DIR
+        os.system(del_logs_cmd)
+
+    os.makedirs(LOGS_DIR, exist_ok=True)
+
+    # Setting up the node environment and clearing cache.
+    utils.set_permissions()
+    utils.set_http_proxies()
```

Comment:
you don't need this. all machines will have this.

Reply:
removed